Physical_Engine/Query/SolidVolume.cs

```diff
@@ -85,15 +81,12 @@ public static double SolidVolume(this oM.Physical.Elements.ISurface surface)
 
             if (surface.Offset != Offset.Centre && !surface.Location.IIsPlanar())
                 Base.Compute.RecordWarning("The SolidVolume for non-Planar ISurfaces with offsets other than Centre is approxamite at best");
-
             double area = surface.Location.IArea();
             area -= surface.Openings.Sum(x => x.Location.IArea());
-            
-            return area * surface.Construction.IThickness();
+            return area * surface.Construction.IVolumePerArea();
```

Comment:
This is a good question... Depends on how we look at the panel: whether we want to get its outer volume or the actual volume of solids included, skipping voids. I do not know the answer to this question tbh

Reply:
This aligns with the work we've already done in the Structural space, replacing AverageThickness with VolumePerArea and having a separate method TotalThickness which gets the outer extent of the panel property

This is chiefly useful for LCA work, material takeoff, and visualization, as Revit (for now) is going to ignore the material composition and just find a wall/floor type with the same name, so maintaining both volume (for material takeoff) and thickness (for display purposes) makes sense- I can't think of a better rule without implementing ribbed, waffle, etc. in the Physical space (which should eventually happen)

Reply:
I would say we are dealing with a philosophical question here, which I personally cannot answer: both concepts of volume are correct, one included nested voids, the other not. So I cannot tell whether this change is right or wrong. Not sure if others can, so I would be fine with merging your proposal under the condition that it will be re-reviewed by @IsakNaslundBh as our main structural philosopher once he is back, before the Beta release. What do you think?